src/test/fixtures/composes.ts

```diff
@@ -645,6 +689,49 @@ export const mockStatus = (composeId: string): ComposeStatus => {
         ],
       },
     },
+    '1679d95b-8f1d-4982-8c53-8c2afa4ab04c': {
+      image_status: {
+        status: 'success',
+        upload_status: {
+          options: {
+            url: 'https://s3.amazonaws.com/1679d95b-8f1d-4982-8c53-8c2afa4ab04c-disk.qcow2',
+          },
+          status: 'success',
+          type: 'aws.s3',
+        },
+      },
+      request: {
+        distribution: RHEL_8,
+        image_requests: [
+          {
+            architecture: 'x86_64',
+            image_type: 'aws',
+            upload_request: {
+              type: 'aws.s3',
+              options: {},
+            },
+          },
```

Comment:
Last one, I promise :no_mouth: 

```suggestion
          {
            architecture: 'x86_64',
            image_type: 'guest-image',
            upload_request: {
              type: 'aws.s3',
              options: {},
            },
          },
```

Reply:
ahahah :smile: I'm glad you catching them all!